ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -378,32 +391,78 @@ class ServicesHomeScreen extends React.Component<Props, State> {
 const mapStateToProps = (state: GlobalState) => {
   const { services } = state.entities;
 
+  const potUserMetadata = userMetadataSelector(state);
+  const userMetadata = pot.getOrElse(potUserMetadata, undefined);
+
   const isAnyServiceLoading =
     Object.keys(services.byId).find(k => {
       const oneService = services.byId[k];
       return oneService !== undefined && pot.isLoading(oneService);
     }) !== undefined;
 
   const isLoading =
-    pot.isLoading(state.entities.services.visible) || isAnyServiceLoading;
+    pot.isLoading(state.entities.services.visible) ||
+    isAnyServiceLoading ||
+    pot.isLoading(potUserMetadata);
 
   return {
     allOrganizations: lexicallyOrderedAllOrganizations(state),
     isLoading,
-    organizationsSelected: organizationsFiscalCodesSelectedStateSelector(state)
+    organizationsSelected: organizationsOfInterestSelector(state),
+    userMetadata
   };
 };
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
   refreshServices: () => dispatch(loadVisibleServices.request()),
-  saveSelectedOrganizationItems: (selectedItemIds: Option<Set<string>>) => {
+  saveSelectedOrganizationItems: (
+    userMetadata: UserMetadata,
+    selectedItemIds: Option<Set<string>>
+  ) => {
     if (selectedItemIds.isSome()) {
-      dispatch(setSelectedOrganizations(Array.from(selectedItemIds.value)));
+      const metadata = userMetadata.metadata;
+      dispatch(
+        userMetadataUpsert.request({
+          ...userMetadata,
+          // tslint:disable-next-line: no-useless-cast
+          version: (userMetadata.version as number) + 1,
+          metadata: {
+            ...metadata,
+            organizationsOfInterest: Array.from(selectedItemIds.value)
+          }
+        })
+      );
     }
   }
 });
 
+const mergeProps = (
+  stateProps: ReturnType<typeof mapStateToProps>,
+  dispatchProps: ReturnType<typeof mapDispatchToProps>,
+  ownProps: OwnProps
+) => {
+  const dispatchUpdateOrganizationsOfInterestMetadata = (
+    selectedItemIds: Option<Set<string>>
+  ) => {
+    if (selectedItemIds.isSome() && stateProps.userMetadata) {
```

Comment:
I have a doubt about the value returned by [fetchUserMetadata](fetchUserMetadata). Maybe @francescopersico  could help us: when the backend responses with **_status===204_** (no content, for example: the first fetch for a user who has no metadata) an error will be returned. In this way, the _userMetaData_ will never be updated cause the _pot_ is in error status. I guess, in that specific case, we should provide an empty _userMetaData_ object

Reply:
within https://github.com/teamdigitale/io-app/pull/1260/commits/b8e42858e791788d952f236c78f17fe857d9e997 a mocked object for `userMetadata` is created to avoid errors on the request to backend. 
```
noneUserMetadata: UserMetadata = {
  version: 0,
  metadata: {
    experimentalFeatures: undefined,
    organizationsOfInterest: undefined
  }
};
```
If the load fails, it should be disabled the possibility of change the user selection. Having many components being rearranged within https://github.com/teamdigitale/io-app/pull/1269, it could be managed once the pr is closed (here the related story: https://www.pivotaltracker.com/story/show/168312476)